airflow/api_fastapi/core_api/openapi/v1-generated.yaml

```diff
@@ -6711,12 +6711,16 @@ paths:
       summary: Get Log
       description: Get logs for a specific task instance.
       operationId: get_log
+      security:
+      - OAuth2PasswordBearer: []
       parameters:
       - name: dag_id
         in: path
         required: true
         schema:
-          type: string
+          anyOf:
```

Comment:
This part of the spec should not be updated.

https://github.com/apache/airflow/pull/47485 Fixes it and has been merged, do you mind rebasing, this change should disappear.